src/main/kotlin/no/nav/syfo/behandlendeenhet/api/internad/BehandlendeEnhetAPI.kt

```diff
@@ -58,17 +60,49 @@ fun Route.registrerPersonApi(
                 token = token,
             )
 
-            val body = call.receive<PersonDTO>()
+            val body = call.receive<BehandlendeEnhetDTO>()
 
             val oppfolgingsenhet = enhetService.updateOppfolgingsenhet(
                 callId = callId,
                 personIdent = PersonIdentNumber(body.personident),
-                isNavUtland = body.isNavUtland,
+                enhet = body.oppfolgingsenhet?.let { Enhet(it) },
                 veilederToken = token,
             )
 
             if (oppfolgingsenhet != null) {
-                call.respond(oppfolgingsenhet.toPersonDTO())
+                call.respond(oppfolgingsenhet.toBehandlendeEnhetDTO())
+            } else {
+                log.error("Could not set oppfolgingsenhet in database")
+                call.respond(HttpStatusCode.BadRequest)
+            }
+        }
+        post(internadBehandlendeEnhetApiV2PersonsPath) {
```

Comment:
Litt usikker på om vi skal ha et endepunkt for å sette oppfølgingsenhet for mange personer på en gang. Har derfor ikke laget tester for denne ennå.

Reply:
Jeg synes at vi bare kan bør gå for minste løsning og begrense det til å tildele én og én i første omgang 😊

Reply:
Jeg tenkte initielt at `/persons` er det vi kommer til å kalle fra begge frontendene, og at fra modia-enkeltperson så sender man bare en liste på én person. Det må jo gjøres i tre steg uansett, men vi kan godt starte med `/person` hvis vi tenker at det kommer til å leve litt i prod kun i syfomodiaperson.

Reply:
Tar det bort i første omgang, men vi kan godt utvide i en senere PR.